rllib/agents/marwil/marwil_tf_policy.py

```diff
@@ -1,6 +1,7 @@
 import logging
 import gym
 from typing import Optional, Dict
+import sys
```

Comment:
Can we do the same in `marwil_torch_policy.py`?

Reply:
done